Update

Bashamega opened the pull request:
Hello:)
I have added the google reCAPTCHA, and fixed the typos, but the author section is still not working.
Note: Please replace the token with your token, I don't have a token, so I did not test it but it should work

gabrysia694 commented:
> Hello:) I have added the google reCAPTCHA, and fixed the typos, but the author section is still not working. Note: Please replace the token with your token, I don't have a token, so I did not test it but it should work

Hi, do you have any idea why the author section is not working?

Bashamega commented:
> > Hello:) I have added the google reCAPTCHA, and fixed the typos, but the author section is still not working. Note: Please replace the token with your token, I don't have a token, so I did not test it but it should work
> 
> Hi, do you have any idea why the author section is not working?

Hello:) I don't have any ides why it is not working. I think it could be another issue.
Please approve this.

Bashamega commented:
@gabrysia694
